### ISSUE TITLE
Add `<html>` as node

### ISSUE THREAD

**Issue by getreu** [izyuumi/html2md-rs#16]
Is the root node `<html>` ... `</html>` recognized?

**Comment by izyuumi** [comment]
Good point. Since this library was made for [LAME](https://github.com/izyuumi/LAME), I did not add HTML. It will come in the next update.

**Comment by izyuumi** [comment]
It will be released with v0.7.0.

**Comment by getreu** [comment]
I did a quick test with 0.7.0 and this input:

```html
<meta http-equiv="content-type" content="text/html; charset=utf-8"><a class="_hero-button_slqo1c _yellow-button_g91299 _button_g91299" href="https://doc.rust-lang.org/cargo/getting-started/installation.html"><svg width="14" height="17" viewBox="0 0 14 17" fill="currentColor" xmlns="http://www.w3.org/2000/svg" class="_icon_slqo1c"></svg></a><a class="_hero-button_slqo1c _yellow-button_g91299 _button_g91299" href="https://doc.rust-lang.org/cargo/getting-started/installation.html">
    Install Cargo
  </a>
```

Produces the incorrect errror:

    Error Malformed attribute: utf-8" - Missing quotation mark at around index

**Comment by izyuumi** [comment]
Could you create a new issue for this? I know why it’s happening and it’s not related to the html tag alone. Thank you!

**Comment by izyuumi** [comment]
This should be resolved in v0.7.1.
